internal/cmd/operator-sdk/generate/bundle/cmd.go

```diff
@@ -112,16 +113,29 @@ func (c *bundleCmd) addFlagsTo(fs *pflag.FlagSet) {
 
 	fs.StringVarP(&c.version, "version", "v", "", "Semantic version of the operator in the generated bundle. "+
 		"Only set if creating a new bundle or upgrading your operator")
-	fs.StringVar(&c.inputDir, "input-dir", "", "Directory to read an existing bundle from. "+
-		"This directory is the parent of your bundle 'manifests' directory, and different from --deploy-dir")
+	fs.StringVar(&c.inputDir, "input-dir", "", "Directory to read cluster-ready operator manifests from. "+
```

Comment:
i forget which of these is required, but isn't there a flag you can throw here that marks them as such in the help text?

Reply:
None of these flags are always required, only conditionally required based on other input data.